static/app/views/dashboardsV2/widgetCard/chart.tsx

```diff
@@ -43,6 +43,7 @@ import {DisplayType, Widget, WidgetType} from '../types';
 
 import WidgetQueries from './widgetQueries';
 
+const OTHER = 'Other';
```

Comment:
can "other" be in other languages too?

Reply:
Good catch! It turns out that in this case we don't translate it from the API, so adding i18n here actually breaks the desired functionality.